core/src/main/java/io/ddf/DDF.java

```diff
@@ -83,6 +85,20 @@ public void setIsDDFView(boolean isDDFView) {
     this.mIsDDFView = isDDFView;
   }
 
+  public boolean isCache() {
+    return this.isCache;
+  }
+
+  public void cache() {
+    this.isCache = true;
+    this.getRepresentationHandler().cache(false);
```

Comment:
What happens when this throw an exception? How about when it's called by multiple user? Maybe there should be a lock on `isCached`?